modules/backup-hosted-cluster-virt.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assembly:
+//
+// * hosted_control_planes/hcp_high_availability/hcp-backup-restore-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="backup-hosted-cluster-virt_{context}"]
+= Backing up a hosted cluster on {VirtProductName}
+
+When you back up a hosted cluster on {VirtProductName}, the hosted cluster can remain running. The backup contains the hosted control plane components, the etcd for the hosted cluster, and the data that is stored in the persistent volume claims (PVCs) for the hosted cluster. The backup does not contain any KubeVirt virtual machines (VMs) that are used as compute nodes. Those VMs are automatically re-created after the restore process is completed.
```

Comment:
note, the PVCs for the hosted cluster are only backed up when the HCP OpenShift Virtualization cluster is running on local infrastructure. Meaning that the HCP worker nodes are running in the same namespace as the HCP pods.

This could be clarified by saying something like

```
The backup contains the hosted control plane components and the etcd for the hosted cluster. Hosted cluster workload data stored in persistent volume claims (PVCs) that are provisioned by KubeVirt CSI will also be backed up when the hosted cluster is not running worker nodes on external infrastructure.
```